gimme

```diff
@@ -49,9 +49,9 @@ set -o pipefail
 
 [[ ${GIMME_DEBUG} ]] && set -x
 
-GIMME_VERSION="v1.2.0"
-GIMME_COPYRIGHT="Copyright (c) 2016 Dan Buch, Tianon Gravi, Travis CI GmbH"
-GIMME_LICENSE_URL="https://raw.githubusercontent.com/travis-ci/gimme/v1.2.0/LICENSE"
+GIMME_VERSION="v1.3.0"
+GIMME_COPYRIGHT="Copyright (c) 2018 Dan Buch, Tianon Gravi, Travis CI GmbH"
```

Comment:
This should be a range, including the first year, to establish when it was first released, for precedence.

Reply:
TIL!